lib/ibm_watson/tone_analyzer_v3.rb

```diff
@@ -97,7 +97,7 @@ def initialize(args = {})
     #########################
 
     ##
-    # @!method tone(tone_input:, content_type:, sentences: nil, tones: nil, content_language: nil, accept_language: nil)
+    # @!method tone(tone_input:, content_type: nil, sentences: nil, tones: nil, content_language: nil, accept_language: nil)
```

Comment:
Can you check to make sure content type can be optional here?

Reply:
Looks like it is optional.